command/delete/delete.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { tryDelete } = require('../../lib/discordjs-utils');
+const { deleteShortcut, getShortcut } = require('../../lib/shortcuts');
+
+const info = {
+  name: 'Delete',
+  command: 'delete',
+  args: 'name *',
+  description: 'Remove a custom reaction shortcut'
+};
+
+const action = async (message, args) => {
+  const name = args[0];
+  const dm = await message.author.createDM();
+  const author = message.author;
+  if (!name) {
+    dm.send('Error: No name provided');
+    return;
+  }
+  if (!getShortcut(author.id, name)) {
+    dm.send(`Error: No shortcut corresponding to ${name}`);
+    return;
+  }
```

Comment:
I think it would be better to move this into `deleteShortcut`.
deleteShortcut should throw an error if the shortcut does not exists.

Reply:
I made the change, it's better